cli/src/commands/bookmark/create.rs

```diff
@@ -51,6 +53,14 @@ pub fn cmd_bookmark_create(
     args: &BookmarkCreateArgs,
 ) -> Result<(), CommandError> {
     let mut workspace_command = command.workspace_helper(ui)?;
+    // TODO: Make args.revision required in jj 0.32+. See https://github.com/jj-vcs/jj/issues/5374.
```

Comment:
Sorry, I miss-placed my nit before but could you move the comment below the doc comment for the specific arg. Thanks.

Reply:
Done.